### ISSUE TITLE
Does version 1.4.1 in NPM have latest scroll changes?

### ISSUE THREAD

**Issue by juancarlosfarah** [rohanchandra/react-terminal-component#26]
I've installed the latest version (v1.4.1), but I still see the code below and not the change in PR #17.

```
{
  key: "componentDidUpdate",
  value: function componentDidUpdate() {
    var autoFocus = this.props.autoFocus;

    if (this.inputRef) {
      this.inputRef.scrollIntoView();
    }
    ...
}
```

Has that change been released?

**Comment by juancarlosfarah** [comment]
I just downloaded the package directly from `npm` using `npm pack react-terminal-component` and upon inspection, it looks like the changes were not included. Not sure if I'm missing something.

**Comment by juancarlosfarah** [comment]
@rohanchandra, sorry to insist with this. I just wanted to know if there will be a new deployment to NPM to fix this. Thanks.

**Comment by rohanchandra** [comment]
Released in `react-terminal-component@1.5.0`
